Makefile

```diff
@@ -50,6 +50,9 @@ update: deps api api-docs app-sre-saas-template
 verify: update staticcheck fmt vet promtool
 	git diff-index --cached --quiet --ignore-submodules HEAD --
 	git diff-files --quiet --ignore-submodules
+	git diff --exit-code HEAD -- > "$(ARTIFACT_DIR)/x.patch"
+	$(eval STATUS = $(shell sh -c 'git diff --exit-code HEAD -- | base64 -w0'))
+	$(if $(strip $(STATUS)),$(error Git diff detected changes: ${STATUS}))
```

Comment:
We'll want to drop this cruft that helps me update autogenerated content once folks feel like the rest of the changes look good.  It would be awesome if there was a change we could actually land to expose this diff in CI artifacts, but even if someone figures that out, it would be orthogonal work for a separate pull request.

/hold

Reply:
I've dropped the git-diff WIP commit with 27b2a8a9 -> 97445d76, now that I seem to have everything we need.

/hold cancel